api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceAssertionUtils.java

```diff
@@ -38,12 +37,18 @@ public static void assertClaimantMatchesClaimantDTO(ClaimantDTO claimant, Claima
         assertAddressEqual(persistedClaim.getAddress(), claimant.getAddress());
     }
 
-    public static RequestEntity buildClaimRequestEntity(Object requestObject) {
+    public static RequestEntity buildCreateClaimRequestEntity(Object requestObject) {
         var headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
         return new RequestEntity<>(requestObject, headers, HttpMethod.POST, CLAIMANT_ENDPOINT_URI_V3);
     }
 
+    public static RequestEntity buildRetrieveClaimRequestEntity(UUID claimId) {
+        var headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
```

Comment:
Can we refactor the header creation into a method which can be reused in these two methods, it can then be inlined in the return statement which makes these much more concise and removes the duplication.